python/ray/serve/storage/tests/test_kv_store.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pytest
+from ray.serve.storage.kv_store import (RayInternalKVStore, RayLocalKVStore,
+                                        RayS3KVStore)
+from ray.serve.tests.conftest import (  # noqa: F401 imported but unused
+    serve_instance, shared_serve_instance)
+
+
+def test_ray_internal_kv(serve_instance):  # noqa: F811
+    with pytest.raises(TypeError):
+        RayInternalKVStore(namespace=1)
+        RayInternalKVStore(namespace=b"")
+
+    kv = RayInternalKVStore()
```

Comment:
FYI you'll need to add this to the bazel BUILD file in serve root for it to run in CI